open-metadata-implementation/admin-services/admin-services-client/src/main/java/org/odpi/openmetadata/adminservices/client/OMAGServerPlatformConfigurationClient.java

```diff
@@ -269,7 +269,7 @@ public Connection getPlatformSecurityConnection() throws OMAGNotAuthorizedExcept
                                                                    OMAGConfigurationErrorException,
                                                                    OMAGInvalidParameterException {
         final String methodName  = "getPlatformServerConfigs";
-        final String urlTemplate = "/open-metadata/admin-services/users/{0}/configurations";
+        final String urlTemplate = "/open-metadata/admin-services/users/{0}/platform/server-configurations";
```

Comment:
should not have platform in the name as the server configuration documents are not associated with a platform

Reply:
removed the platform